packages/runtime/src/account.ts

```diff
@@ -216,11 +219,12 @@ class App {
   readonly id: number;
   readonly attributes: SSCAttributesM;
 
-  constructor (appId: number, params: SSCDeploymentFlags) {
+  constructor (appId: number, params: SSCDeploymentFlags,
```

Comment:
Please add a a documentation. 
Explain that approvalProgram and clearProgram must be a TEAL code (someone could think it's a path to a file or a compiled code).

Reply:
Ah, yes.